blogs/scripts/scripts.js

```diff
@@ -591,6 +718,7 @@ async function loadLazy(doc) {
 
   const header = doc.querySelector('header');
   const footer = doc.querySelector('footer');
+  await loadKeysightHeader(header);
   loadHeader(header);
```

Comment:
is there a reason why we are loading 2 headers?

Also, any reason that the keysight header isn't a block?